model/build.gradle

```diff
@@ -6,6 +6,7 @@ dependencies {
     annotationProcessor 'com.google.auto.value:auto-value:1.7.4'
     compileOnly 'com.google.auto.value:auto-value-annotations:1.7.4'
     implementation 'org.jetbrains:annotations:20.1.0'
+    implementation 'com.google.code.findbugs:jsr305:3.0.2'
```

Comment:
question: why do we need to import this library?

Reply:
The @Nullable annotation is not included in the JDK by default so it has to be included via a dependency.  `com.google.code.findbugs:jsr305` is actually where we've been getting this annotation in `core` and `main`, but it was a transitive dependency via `com.google.guava:guava`.  Since `model` doesn't need Guava, I just imported the dependency directly.